src/aat/resources/features/F-049/S-231.td.json

```diff
@@ -0,0 +1,27 @@
+{
+  "_guid_": "S-231",
+  "_extends_": "F-049_Test_Data_Base",
+  "title": "must return 403 when request provide invalid authentication credentials",
+  "specs": [
+    "a detailed profile in CCD",
+    "Start case creation as Case worker",
+    "CCD Data Store",
+    "provide invalid authentication credentials",
+    "code is HTTP-403"
+  ],
+  "request": {
+    "pathVariables": {
+      "uid": "dummy"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_403_Response",
+    "body" : {
+      "timestamp" : "[[ANY]]",
+      "status" : 403,
+      "error" : "Forbidden",
+      "message" : "Access Denied",
+      "path" : "[[ANY]]"
```

Comment:
all these fields except for message are common, and inherited from parent (Common_403_Response)